lib/bin.ts

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { resolve } from 'path';
+import { parseArgs } from '@pkgjs/parseargs';
+import { runTests } from './runTest';
+
+const options = {
+	'vscode-executable-path': { type: 'string' },
+	version: { type: 'string' },
+	platform: { type: 'string' },
+	'reuse-machine-install': { type: 'boolean' },
+	'extension-development-path': { type: 'string', multiple: true },
+	'launch-args': { type: 'string', multiple: true },
+	'extract-sync': { type: 'boolean' },
+	'help': { type: 'boolean' },
+} as const;
+
+const help = `Usage: code-test [options][extension-tests-path...]
+
+Options
+	--vscode-executable-path      The VS Code executable path used for testing.
+	--version                     The VS Code version to download.
```

Comment:
This flag is a bit weird IMO. `--version` usually prints the version of a tool, but in this case it matches the `version` option of `runTests`. Any suggestions?

Reply:
I would probably rename it to `--vscode-version`. PS: I'm not a maintainer.

Reply:
I agree. I’m glad to see someone mention this, regardless whether you’re a maintainer or not. :)

I changed it to `--vscode-version`, and repurposed the `--version` flag to print version information.